Library/Homebrew/utils/curl.rb

```diff
@@ -163,12 +184,27 @@ def curl_with_workarounds(
       result
     end
 
-    def curl(*args, print_stdout: true, **options)
-      result = curl_with_workarounds(*args, print_stdout: print_stdout, **options)
+    sig {
+      params(
+        args:    T.untyped,
+        options: T.untyped,
+      ).returns(SystemCommand::Result)
+    }
+    def curl(*args, **options)
```

Comment:
@MikeMcQuaid I wanted to avoid carrying `print_stdout` in annotations for subclasses and moved it inside the method. Was it the right solution?

Reply:
@Kentzo Please extract this into a separate PR too.